src/apify_helpers.js

```diff
@@ -228,13 +258,14 @@ const getActorAdditionalFields = async (z, bundle) => {
             type: 'integer',
         },
         {
-            // TODO: This should be a drop-down with allowed memory sizes, using values from apify-shared/contants
             label: 'Memory',
             helpText: 'Amount of memory allocated for the actor run, in megabytes. The more memory, the faster your actor will run.',
             key: 'memoryMbytes',
             required: false,
-            default: memoryMbytes || 1024,
-            type: 'integer',
+            // NOTE: Zapier UI allows only choices with strings
+            default: (memoryMbytes || 1024).toString(),
```

Comment:
1024 should be a constant. Shouldn't we go higher? 2GB?

Reply:
Assuming that people will quite often user puppeteer so 2GB would make sense. Or maybe better the default would be empty and you can override tasks/actor default value only if you want.

Reply:
Either way, the number should be a CONSTANT.